contrib/ubase/Makefile

```diff
@@ -0,0 +1,133 @@
+TOPDIR = $(realpath ../..)
+
+
+HDR = \
+	arg.h \
+	compat.h \
+	crypt.h \
+	fs.h \
+	md5.h \
+	queue.h \
+	sha1.h \
+	sha224.h \
+	sha256.h \
+	sha384.h \
+	sha512.h \
+	sha512-224.h \
+	sha512-256.h \
+	text.h \
+	utf.h \
+	util.h
+
+LIBUTILSRC = \
+	libutil/agetcwd.c        \
+	libutil/agetline.c       \
+	libutil/apathmax.c       \
+	libutil/concat.c         \
+	libutil/ealloc.c         \
+	libutil/eprintf.c        \
+	libutil/estrtol.c        \
+	libutil/estrtoul.c       \
+	libutil/explicit_bzero.c \
+	libutil/passwd.c         \
+	libutil/proc.c           \
+	libutil/putword.c        \
+	libutil/recurse.c        \
+	libutil/strlcat.c        \
+	libutil/strlcpy.c        \
+	libutil/strtonum.c       \
+	libutil/tty.c
+
+BIN = \
+	clear             \
+	getty							\
+	id                \
+	login             \
+	nologin           \
+	pagesize          \
+	respawn           \
+	stat              \
+	su                \
+	truncate          \
+	watch
+
+BIN_DONT_WORK = \
+	blkdiscard        \
+	chvt              \
+	ctrlaltdel        \
+	dd                \
+	df                \
+	dmesg             \
+	eject             \
+	fallocate         \
+	free              \
+	freeramdisk       \
+	fsfreeze          \
+	halt              \
+	hwclock           \
+	insmod            \
+	killall5          \
+	last              \
+	lastlog           \
+	lsmod             \
+	lsusb             \
+	mesg              \
```

Comment:
What stops us from porting `mesg` tool?

Reply:
I added this tool (https://github.com/cahirwpz/mimiker/pull/1371/commits/4c69c2dd6880688b0b34c2eb726850afc236c9ff); it required a small fix in the source code (https://github.com/cahirwpz/mimiker/pull/1371/commits/c200ca70abccd5599d291ef08b3c0ad2e18e3ae8) due to `-Wsometimes-uninitialized` error.